limbus/components/base.py

```
@@ -73,7 +73,14 @@ def forward(self) -> ComponentState:  # noqa: D102
 
 
 class ImageShow(Component):
-    """Component to show the input image."""
+    """Component to show images.
+
+    Args:
+        name: component name.
+        window: name of the window containing the image/s.
+                Default: None -> window name is the component id.
+
+    """
```

add at some place that the component forward should return an exception when there is an error.

---

think about errors in the constructor